core/src/navigation/LeftNav.html

```diff
@@ -209,6 +221,9 @@
 
 <style type="text/scss">
   $topNavHeight: 48px;
+  $footerHeight: calc(0.85714rem + (2 * 13px) + 3px);
```

Comment:
I did it this way because I wanted to give a *clear* information how this height is calculated. 
`0.85714rem ` is the _font-size_ value for Fundamental's _minus-1_ font. 
In fact, I could use one fixed magic number and I wouldn't have any scrolling issue ;)

On the other hand, as you noticed, it would be nice to use variables. To make it sensible, I'd have to get them from Fundamental which means importing another SCSS modules. It's pointless in my opinion. Also, declaring those variables manually gives no advantage over a simple comment line which explains everything.